source/code/scxsystemlib/disk/staticphysicaldiskenumeration.cpp

```diff
@@ -119,7 +137,15 @@ namespace SCXSystemLib
         }
 #endif
 
-        m_deps->RefreshMNTTab();
+        RefreshMNTTabParam *param = NULL;
+        if ( device != L"" ) param = new RefreshMNTTabParam(RefreshMNTTabParam::DEVICE, device);
+        m_deps->RefreshMNTTab(param);
+        if ( param ) free(param);
+
+#if defined(aix) || defined(hpux)
```

Comment:
is hpux check really required. We no longer support hpux

Reply:
Hpux is not supported, But I have done this for completeness